app/src/main/java/com/serranoie/app/itinero/ui/MainActivity.kt

```
@@ -27,34 +26,52 @@ import androidx.compose.foundation.layout.imePadding
+import com.serranoie.itinero.core.data.remote.UnauthorizedException
+import com.serranoie.itinero.core.data.remote.UnauthorizedHandler
+import com.serranoie.itinero.core.domain.repository.AuthRepository
+import com.serranoie.itinero.core.domain.usecase.TravelUseCase
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
 import org.koin.android.ext.android.inject
 import org.koin.androidx.compose.KoinAndroidContext
 
 class MainActivity : ComponentActivity() {
     private lateinit var navController: NavHostController
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)